<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-S76SA9N" failures="1" tests="5" name="runner.approval.HRISNotesRunner" time="169.931" errors="0" timestamp="2022-07-12T09:39:24 ICT" skipped="0">
  <testcase classname="runner.approval.HRISNotesRunner" name="runScenario" time="35.696">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at definitions.data.HRISBukcetDataDefine.validateApprovalNotes(HRISBukcetDataDefine.java:736)
at âœ½.User validate approval notes "reject" button(file:///D:/Project/Java/TestNG/HRIS/src/test/resources/features/003-MODUL%20APPROVAL/HRISNotes.feature:45)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.approval.HRISNotesRunner" name="runScenario" time="29.323"/>
  <system-out/>
  <testcase classname="runner.approval.HRISNotesRunner" name="runScenario" time="33.452"/>
  <system-out/>
  <testcase classname="runner.approval.HRISNotesRunner" name="runScenario" time="38.902"/>
  <system-out/>
  <testcase classname="runner.approval.HRISNotesRunner" name="runScenario" time="32.558"/>
  <system-out/>
</testsuite> <!-- runner.approval.HRISNotesRunner -->
